src/Symfony/Component/WebLink/Tests/LinkTest.php

```diff
@@ -54,11 +55,11 @@ public function testMultipleRels()
         $link = (new Link())
             ->withHref('http://www.google.com')
             ->withRel('next')
-            ->withRel('reference');
+            ->withRel('acl');
```

Comment:
the fact that the test using `acl` as relation broke and you had to change it is another proof that the enum might be a bad idea (and that it is actually a BC break)

Reply:
I agree with your comment about the BC.
I mentioned the problem in a comment above.

I was waiting for a feedback to see how to deal with it.
Leaving a string array would then allow not to get stuck on a closed list.